spec/controllers/api/v1/concepts_controller_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+
+describe Api::V1::ConceptsController, type: :controller do
+
+  context 'POST #create' do
+    let!(:user) {FactoryGirl.create(:admin)}
+    let!(:token) { double :acceptable? => true, :resource_owner_id => user.id }
+    let!(:concept_name) {'Concept1'}
+    let!(:parent_concept) {FactoryGirl.create(:concept)}
+
```

Comment:
@kriskelly no worries, I appreciate the minor nitpicks so keep 'em coming. Sounds like a good idea, perhaps we can get rubocop involved...